lib/lighthouse/benefits_documents/form526/update_documents_status_service.rb

```diff
@@ -66,34 +67,29 @@ def update_document_status(status)
         document_upload = @lighthouse526_document_uploads.find_by!(lighthouse_document_request_id: status['requestId'])
         statsd_document_base_key(STATSD_DOCUMENT_TYPE_KEY_MAP[document_upload.document_type])
 
-        status_updater(status, document_upload)
-        @status_updater.update_status
+        status_updater = BenefitsDocuments::Form526::UploadStatusUpdater.new(status, document_upload)
+        status_updater.update_status
 
         if document_upload.completed?
           # ex. 'api.form526.lighthouse_document_upload_processing_status.bdd_instructions.complete'
           StatsD.increment("#{@statsd_document_base_key}.#{STATSD_DOCUMENT_COMPLETE_KEY}")
         elsif document_upload.failed?
-          log_failure(document_upload)
-        elsif @status_updater.processing_timeout?
+          log_failure(status_updater, document_upload)
+        elsif status_updater.processing_timeout?
           # Triggered when a document is still pending more than 24 hours after processing began
           # ex. 'api.form526.lighthouse_document_upload_processing_status.bdd_instructions.processing_timeout'
           StatsD.increment("#{@statsd_document_base_key}.#{STATSD_PROCESSING_TIMEOUT_KEY}")
         end
       end
 
-      def status_updater(status, document_upload)
-        # UploadStatusUpdater encapsulates all parsing of a status response from Lighthouse
```

Comment:
Do you want to preserve this docstring in the correction?

Reply:
Ah I mean to but nah we have docs in the class that explain what it does I was a bit heavy handed with the docstrings in the original version of this